docs/integrations/destinations/gcs.md

```diff
@@ -0,0 +1,154 @@
+# Google Cloud Storage
+
+## Overview
+
+This destination writes data to GCS bucket.
+
+The Airbyte GCS destination allows you to sync data to cloud storage buckets. Each stream is written to its own directory under the bucket.
+
+## Sync Mode
+
+| Feature | Support | Notes |
+| :--- | :---: | :--- |
+| Full Refresh Sync | ✅ | Warning: this mode deletes all previously synced data in the configured bucket path. |
+| Incremental - Append Sync | ✅ | |
+| Namespaces | ❌ | Setting a specific bucket path is equivalent to having separate namespaces. |
+
+## Configuration
+
+| Parameter | Type | Notes |
+| :--- | :---: | :--- |
+| GCS Project ID | string | Project ID |
+| GCS Bucket Name | string | Name of the bucket to sync data into. |
+| GCS Bucket Path | string | Subdirectory under the above bucket to sync the data into. |
+| GCS Region | string | See [here](https://cloud.google.com/storage/docs/locations) for all region codes. |
+| Access Key ID | string | GCP HMAC credential. |
+| Secret Access Key | string | GCP HMAC credential. |
+| Format | object | Format specific configuration. See below for details. |
+
+⚠️ Please note that under "Full Refresh Sync" mode, data in the configured bucket and path will be wiped out before each sync. We recommend you to provision a dedicated S3 resource for this sync to prevent unexpected data deletion from misconfiguration. ⚠️
```

Comment:
```suggestion
⚠️ Please note that under "Full Refresh Sync" mode, data in the configured bucket and path will be wiped out before each sync. We recommend you to provision a dedicated GCS bucket for this sync to prevent unexpected data deletion from misconfiguration. ⚠️
```